src/main/java/application/domain/element/LadderElement.java

```diff
@@ -0,0 +1,17 @@
+package application.domain.element;
+
+public enum LadderElement {
+    STEP("-----")
+    , EMPTY("     ")
+    , RAIL("|");
```

Comment:
콤마는 한 줄의 끝에 작성해주는 것이 컨벤션입니다.

```suggestion
    STEP("-----"),
    EMPTY("          "),
    RAIL("|");